ca/django_ca/widgets.py

```diff
@@ -103,6 +103,7 @@ class SubjectWidget(CustomMultiWidget):
 
     def __init__(self, attrs: typing.Optional[typing.Dict[str, str]] = None) -> None:
         _widgets = (
+            SubjectTextInput(label=_("Domain Component")),
```

Comment:
This could also be a hidden input field if you prefer to not overwhelm less Active Directory affine users.

Reply:
Hmm. I'm definitely against a hidden field. But one better solution would be to move it below the `Organization` maybe? I don't know exactly where this is supposed to be, I don't know the RFC that much.

Reply:
`DC` is above `OU`s in the hierarchy. Moving it below could cause unexpected issues when sorting; e.g. the first of two `DC` element being shifted in the wrong position, ultimately not respecting the original "logic".

Reply:
well, above `Organization` (as I suggested) would also be above `OrganizationalUnit`. Wouldn't that then work?

Reply:
Ah *I think you're right*. `O` doesn't exist in Active Directory afaik

So yeah that could work.

Reply:
Note that I would not block a merge because of that. If the correct place is above `C`, then so be it. Just let me know what works for you technically.

Reply:
According to IBM's docs it should actually be right after OU: https://www.ibm.com/docs/en/ibm-mq/7.5?topic=certificates-distinguished-names – I adjusted it accordingly.

Looks like quite a useful reference.